src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -473,14 +473,14 @@ private void handleCloudWatchAttemptStatus(CloudWatchAttempt cloudWatchAttempt)
             }
             completedFiles.forEach(file -> {
                 try {
-                    boolean successfullyDeleted = Files.deleteIfExists(file.toPath());
+                    boolean successfullyDeleted = Files.deleteIfExists(Paths.get(file.getSourcePath()));
```

Comment:
We have to remember to avoid using the getSourcePath since it is not reliable with rotations. We chatted about this